testing/tests/DevExpress.ui/collectionWidget.tests.js

```diff
@@ -159,7 +159,8 @@ QUnit.test("custom render func, returns jquery", function(assert) {
             testProp: 5
         }],
         itemTemplate: function(item, index, itemElement) {
-            itemElement.append($("<span />").html("Text is: " + String(item.testProp) + ";"));
+            assert.equal(isRenderer(itemElement), config().useJQueryRenderer, "itemElemenet is correct");
```

Comment:
Why don't you check if the itemElement is the instance of jQuery? It should be more suitable for this case.

Reply:
isRenderer method checks if the itemElement is the instance of jQuery.

Reply:
I know, but `instanceof jQuery` looks more obvious. Isn't it?

Reply:
Yes. But this condition doesn't check if itemElement is dom node. We have to be sure that itemElement isn't DxRenderer also.

DxRenderer doesn't instance of jQuery. So this test will pass without jQuery...

Reply:
Now I got it, thanks.